docs/current/guides/snippets/openshift-gitlab/.gitlab-ci.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+  - build
+
+.dagger:
+  image: golang:1.20
+  tags:
+    - dagger
+  variables:
+    "_EXPERIMENTAL_DAGGER_RUNNER_HOST": "unix:///var/run/dagger/buildkitd.sock"  
```

Comment:
For Dagger Cloud users, do we need to add the below, so that the Cloud token which we have said to create as a secret, is now added to the CI env?

- "DAGGER_CLOUD_TOKEN": $DAGGER_CLOUD_TOKEN